tests/python_package_test/test_engine.py

```diff
@@ -3578,3 +3579,27 @@ def test_boost_from_average_with_single_leaf_trees():
     preds = model.predict(X)
     mean_preds = np.mean(preds)
     assert y.min() <= mean_preds <= y.max()
+
+
+@pytest.mark.parametrize('verbosity_param', lgb.basic._ConfigAliases.get("verbosity"))
+@pytest.mark.parametrize('verbosity', [-1, 0])
+def test_verbosity_can_suppress_alias_warnings(capsys, verbosity_param, verbosity):
```

Comment:
Since the original issue is quite "sensitive", could you please add a test that checks behavior when user passes multiple aliases for the `verbosity` param?
Refer to .